oidc-stub/src/MockOidcAuthorizeHandler.ts

```diff
@@ -0,0 +1,50 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { Logger } from "@aws-lambda-powertools/logger";
+import { Metrics } from "@aws-lambda-powertools/metrics";
+import { Response } from "./utils/Response";
+import { ResourcesEnum } from "./models/enums/ResourcesEnum";
+
+import { HttpCodesEnum } from "./utils/HttpCodesEnum";
+import { LambdaInterface } from "@aws-lambda-powertools/commons";
+import { AppError } from "./utils/AppError";
+import { OidcAuthorizeProcessor } from "./services/OidcAuthorizeProcessor";
+
+
+const POWERTOOLS_METRICS_NAMESPACE = process.env.POWERTOOLS_METRICS_NAMESPACE ? process.env.POWERTOOLS_METRICS_NAMESPACE : Constants.F2F_METRICS_NAMESPACE;
+const POWERTOOLS_LOG_LEVEL = process.env.POWERTOOLS_LOG_LEVEL ? process.env.POWERTOOLS_LOG_LEVEL : Constants.DEBUG;
+const POWERTOOLS_SERVICE_NAME = process.env.POWERTOOLS_SERVICE_NAME ? process.env.POWERTOOLS_SERVICE_NAME : Constants.AUTHORIZATIONCODE_LOGGER_SVC_NAME;
+
+const logger = new Logger({
+	logLevel: POWERTOOLS_LOG_LEVEL,
+	serviceName: POWERTOOLS_SERVICE_NAME,
+});
+
+const metrics = new Metrics({ namespace: POWERTOOLS_METRICS_NAMESPACE, serviceName: POWERTOOLS_SERVICE_NAME });
+
+class MockOidcAutorizeHandler implements LambdaInterface {
+
+	@metrics.logMetrics({ throwOnEmptyMetrics: false, captureColdStartMetric: true })
+	async handler(event: APIGatewayProxyEvent, context: any): Promise<APIGatewayProxyResult> {
+		 switch (event.resource) {
+			case ResourcesEnum.OIDC_AUTHORIZE_ENDPOINT:
+				 if (event.httpMethod === "GET") {
```

Comment:
I don't think we need these 3 lines of logic if each endpoint is pointing to a single lambda?

```
switch (event.resource) {
			case ResourcesEnum.OIDC_AUTHORIZE_ENDPOINT:
				 if (event.httpMethod === "GET") {
```